### ISSUE TITLE
Is there a way to out out of the fancy String diffs?

### ISSUE THREAD

**Issue by benhutchison** [eed3si9n/expecty#116]
I ended up using Expecty as a side-effect of trying Weaver test framework.

```
  test("diff"):
    expect.same("terminated", "terminated 2@35.50 mode=Capture,score=Unknown")
```
Output:

`[error]   terminated  |  terminated[ ][2][@][35][.][50][ ][mode][=][Capture][,][score][=][Unknown]
`
Why does it put all these `[]` s throughout the string diff? Can I opt out of this somehow and just see the raw strings that weren't equal? I often don't find these brackets helpful.

**Comment by Baccata** [comment]
Hi there. Really this should be a discussion on the weaver repo, so I'll answer and close it here. Feel free to open another issue on weaver. 

The tl;dr is that you can create your own assertions in userland. 

Something like : 

``` 
implicit class ExpectExtension(expect: weaver.Expect) {
    def same2(expected: A, found: A)(implicit eqA: Eq[A], showA: Show[A] = Show.fromToString[A], loc: SourceLocation): Expectations = {
      ///
    }   
}
```

Just copy the implementation from [here](https://github.com/disneystreaming/weaver-test/blob/main/modules/core/src/weaver/internals/ExpectSame.scala#L12-L38) and remove the `DiffUtil` part in favour of whatever rendering logic you'd like. 

I'd appreciate a PR that'd change the current rendering in favour of something that'd be closer to Munit, and keep the current rendering in another method : the `DiffUtil` is really useful for comparing strings that only differ by a few characters. Munit is generally better for every other usecase in that regard.

**Comment by BenHutchisonZD** [comment]
Thanks for your advice @Baccata, very helpful

an assertion to my liking
```
object ExpectExt:
  def same[A](expected: A, actual: A)(using loc: SourceLocation, eq: Eq[A] = Eq.fromUniversalEquals[A], s: Show[A] = Show.fromToString[A]): Expectations =
    Expectations(if expected === actual then ().validNel
    else new AssertionException(s"Expected: '${expected.show}'\nActual: '${actual.show}'", NonEmptyList.of(loc)).invalidNel)
```
